### DISCUSSION TITLE
Difficulting connecting with my UDM-SE console

### DISCUSSION THREAD

**Discussion by bubldopl** [NRCHKB/node-red-contrib-unifi-os#55] [General]
Hi, 

Very interesting nodes !

I am trying to connect with my console to test a flow I found in one of your discussions regarding doorbell events on unifi protect. I am relatively familiar with node-red, however not very familiar with the way to enter credentials or access the UDM-SE using the nodes. 

For some reason, I am getting 
 "3/8/2023, 11:49:22 AM[node: Bloken](http://localhost:1880/#) msg : string[17] "Failed to log out""
when trying to run the flow. 

I have configured by UDM-SE access with proper credentials in unifi that have admin privileges on the console. I can ping the console. 

the unifi nodes all appear to be 'initializing'...

any pointer as to where I should look ? 

in my terminal (running node-red on mac) I am getting lots of those errors : 08 Mar 11:59:36 UniFi-Error Bad response from: https://192.168.2.-hidden-:8443/api/auth/login +5s
08 Mar 11:59:36 UniFi-Error Endpoint not found: AxiosError: Request failed with status code 404

Thanks !

**Comment by crxporter** [comment]
The "failed to log out" shouldn't concern you - that's fixed in the newest beta.

The login endpoint error 404 is what I'm worried about - it appears to be failing login.  I see your IP shows "-hidden-" and I'm guessing that's just you editing it for us here? That ought to be your UDM-SE which would live at `192.168.2.1` by default I believe? But the error 404 points to a problem with that IP. The IP for the controller config node should be the same as whatever you use in your browser to connect to the dream machine.

Here's a screenshot of mine (also a UDM-SE):
![image](https://user-images.githubusercontent.com/38265886/223840395-ab83a20e-2f66-485b-861e-8e60ea6ef791.png)

**Reply by bubldopl** [reply]
ok... did a few more tests... got it to work! I have done a complete reinstall of node-red on a separate laptop as I was suspecting some things were broken on my instance. I had tested a few nodes and when installing by mistake node-red-contrib-unificationengine it had crashed node-red, and seemed to be trying to redirect a lot of stuff in the background. I thought I had fixed everything but there seems to be some remaining redirects of some kind preventing access to the url. Anyway, on my 2nd Mac it works ! Now I get to start the fun stuff, my goal being to be able to intercept the ring event and trigger a relay with it. This will avoid running new (stupid) wires to a legacy wireless doorbell. 

Thanks for your quick reply, it helped me look in the right direction !
cheers, Pat

**Reply by crxporter** [reply]
@bubldopl regarding this:

> Now I get to start the fun stuff, my goal being to be able to intercept the ring event and trigger a relay with it. 

how do you feel about almost-ready beta versions? This task is *extremely* simple with the latest beta. I'll send another message in a couple of minutes with the setup...

@marcus-j-davies we should release that soon...

**Reply by crxporter** [reply]
With the latest beta getting a "doorbell ring" event from a unifi camera is as simple as:

1- add a copy of the new `unifi-protect` node
2- configure it to a controller (easiest if you've already set up a controller  config node previously)
3- configure it like this screenshot:
![image](https://user-images.githubusercontent.com/38265886/223875608-08588c02-3968-4e01-b577-a7d5d7f6d989.png)
4- watch the output for bell events

Marcus has done some awesome work taking away a lot of the manual filtering, it's built into the new `unifi-protect` node.

**Comment by bubldopl** [comment]
Wow looks really cool!
I have never used beta code before however, so after many tries to install the packages from the beta files I only managed to break (again) my node-red :-S  I have to work on some stuff tonight but as soon as I have some time (likely tomorrow) I'll be testing it ! Thanks for building theses nodes, I really feel this is an essential component for any unifi system ! 
All the best,
pl
